R/sim_gs_n.R

```diff
@@ -22,18 +22,20 @@
 #' arguments will change as we add additional features.
 #'
 #' @inheritParams sim_fixed_n
-#' @param test A test function such as [wlr()], [maxcombo()], or [rmst()]. The
-#'   simulated data set is passed as the first positional argument to the test
-#'   function provided. Alternatively a list of functions created by
-#'   [create_cutting_test()]. The list form is experimental and currently
-#'   limited. It only accepts one test per cutting (in the future multiple tests
-#'   may be accepted), and all the tests must consistently return the same exact
-#'   results (again this may be more flexible in the future).
-#' @param cutting A list of cutting functions created by [create_cutting()],
-#'   see examples.
+#' @param test One or more test functions such as [wlr()], [maxcombo()], or
+#'   [rmst()]. If a single test function is provided, it will be applied at each
+#'   cut. Alternatively a list of functions created by [create_test()]. The list
+#'   form is experimental and currently limited. It only accepts one test per
+#'   cutting (in the future multiple tests may be accepted), and all the tests
+#'   must consistently return the same exact results (again this may be more
+#'   flexible in the future). Importantly, note that the simulated data set is
+#'   always passed as the first positional argument to each test function
+#'   provided.
+#' @param cut A list of cutting functions created by [create_cut()], see
+#'   examples.
 #' @param seed Random seed.
-#' @param ... Arguments passed to the test function provided by the argument
-#'   `test`.
+#' @param ... Arguments passed to the test function(s) provided by the argument
+#'   `tests`.
```

Comment:
`tests` -> `test` for consistency?

Reply:
Good catch @nanxstats! Fixed in https://github.com/Merck/simtrial/pull/221/commits/d539ae69571753d76e878ec2424ae37424db8c55